jest.config.js

```diff
@@ -162,6 +162,8 @@ module.exports = {
     '<rootDir>/libs/clients/document-provider',
     '<rootDir>/libs/application/templates/login-service',
     '<rootDir>/libs/application/templates/public-debt-payment-plan',
+    '<rootDir>/libs/api/domains/finance',
```

Comment:
You probably want to add codeowners for this two new libs here https://github.com/island-is/island.is/blob/main/.github/CODEOWNERS

Reply:
Yes, good catch. I'll add that.

Reply:
@jeremybarbet this has been added [here](https://github.com/island-is/island.is/pull/4136/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R88). 🙌